tests/acceptance/features/webUIBatchAction/batchAction.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Visibilty of the batch actions menu
```

Comment:
These tests are unrelated to the PR. Please, do this in a separate PR. And then the review process will also be a lot easier.

Reply:
Ok. I will do in next PR